src/dmd/root/stringtable.d

```diff
@@ -40,28 +53,26 @@ struct StringEntry
 struct StringValue
 {
     void* ptrvalue;
-    size_t length;
+    private size_t length;
 
-nothrow:
-pure:
 @nogc:
```

Comment:
Why is this change being made for `pure` and `nothrow` but not for `@nogc`?

Reply:
See the discussion with Marler and Wilzbach above. I had global attributes before but apparently D-style advocates local alphabetically sorted attributes so I changed it. I can go back if you want.

Reply:
I see that the discussion is about local vs. global (I, personally, prefer local), but I don't see anything about treating `@nogc` specially. Am I missing something? What I suggest is that whatever treatment `pure` and `nothrow` get should be the same for `@nogc`.

Reply:
Sorry I completely misread your comment. I simply forgot to make @nogc local too, thanks for pointing out.